i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

```diff
@@ -17,6 +17,49 @@ Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 22.10.5
+
+Release date: `soon`
+
+> You need to [execute the **configure.sh** script](../graph-views/map-web-install.md#étape-6--exécuter-le-script-configuresh) in order to apply some fixes provided by this release.
+
+#### Bug fixes
+- [API] Fixed an issue with create element in view that could break default icon.
+- [CEIP] Added more details for MAP engine telemetry.
+- [Editor] Fixed an issue in shapes edition modal that caused scrollbar to be reset when browsing images.
+- [Editor] Fixed view edition with some links having corrupted bend points.
+- [Editor] Update draw.io library.
```

Comment:
To rephrase

Reply:
should it be this @nmagniezCentreon ?
Fixed the security issue CVE-2022-42889 (Text4shell).